### PR TITLE
created structured cldf dataset out of Postgres DB

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributors
+
+Name | GitHub user | Description | Role
+---  | ---         | ---         | ---
+Hedvig Skirgård |  | Editor | Author
+Suzanne van der Meer |  | Editor |  Author
+Harald Hammarström |  | Editor | Author
```
Comment (xrotwang) [inline CONTRIBUTORS.md]: I think we should list all names of the citation as authors - in proper order.

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "id": "nts",
+    "title": "Nijmegen Typological Survey",
+    "description": "Dataset on Typological Features, collected 2013-2014 in the Language and Cognition Department at the Max Planck Institute for Psycholinguistics, Max-Planck Gesellschaft, and a European Research Council's Advanced Grant (269484 \"INTERACT\") to Stephen C. Levinson.",
+    "license": "http://creativecommons.org/licenses/by-nc-nd/2.0/de/deed.en",
```
Comment (xrotwang) [inline metadata.json]: We should drop the "ND" clause, because Grambank is - to some extent - derived from NTS.

**Review by xrotwang — CHANGES_REQUESTED** [review]
See comments.

File: metadata.json (hunk 2 above)
Comment (Bibiko) [reply metadata.json]: Do we have permission from the authors to change the copyright?

Comment (xrotwang) [reply metadata.json]: Not explicitly. But given the fact that Grambank is derived (partly) from NTS, and most of the NTS people are also behind Grambank, I'd assume they'd be ok with that.

**Review by xrotwang — APPROVED** [review]

**Pull request merged**